oecologia-australis.csl

```diff
@@ -148,9 +148,9 @@
             </choose>
           </group>
         </if>
-        <else-if type="legislation webpage" match="any">
+        <else-if type="legislation webpage report" match="any">
           <group prefix="(" delimiter=" " suffix=").">
-            <text value="Retrieved on"/>
+            <text value="Accessed in"/>
```

Comment:
"Accessed in" + date is not proper English -- the styleguides seems to suggest: Retrieved, `date`, from `URL` shouldn't we do that rather?

Reply:
@adam3smith in relation to "retrieved on/accessed in" I agree with you. However, I am only changing this style according to the required by Oecologia Australis in-chief editor. I tried to argue in order to follow style guides, but editor preferred on this form.

Reply:
Can you maybe emphasize again that "Accessed in" is *not* a matter of taste. It is simply incorrect English. I'd be very reluctant to accept that. Is the style as requested by the editor documented anywhere?

Reply:
Please do. Both "Accessed + day" and "Accessed on + day" are fine. Accessed in would only work for months, not for specific days.

Reply:
I am waiting for the editorial opinion about this.

Reply:
Editorial board decided for "retrieved in". I will change it.